lib/portlayer/exec/commit.go

```diff
@@ -170,7 +170,7 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 	// best effort update of container cache using committed state - this will not reflect the power on below, however
 	// this is primarily for updating ExtraConfig state.
 	if !creation {
-		defer c.RefreshFromHandle(ctx, h)
+		c.RefreshFromHandle(ctx, h)
```

Comment:
the refreshing is from handle to container base. we'll do refresh after container is started